akka-javasdk/src/main/resources/reference.conf

```diff
@@ -58,17 +58,13 @@ akka.javasdk {
     # the default will anyway not trigger any snapshots)
     snapshot-every = 100
 
-    # When EventSourcedEntity is deleted the existence of the entity is completely cleaned up after this duration..
+    # When a EventSourcedEntity or KeyValueEntity is deleted the existence of the entity is completely cleaned up after
+    # this duration.
```

Comment:
yeah, it would be better to move this to 
```
akka.javasdk.entity.cleanup-deleted-after
```

if we want to keep compatibility (for ese) we could
```
akka.javasdk.entity.cleanup-deleted-after = ${akka.javasdk.event-sourced-entity.cleanup-deleted-after}
```

Reply:
Changed in https://github.com/akka/akka-sdk/pull/289/commits/771b7c379f1e3f2f0d30717b70fd91425a7f8f51